index.php

```diff
@@ -96,6 +97,35 @@
 		header('Content-Type: application/json');
 		echo json_encode($list, JSON_PRETTY_PRINT);
 
+	} elseif($action === 'detect' && !empty($url)) {
+
+		$format = $params['format']
+			or returnClientError('You must specify a format!');
+
+		foreach(Bridge::listBridges() as $bridgeName) {
+
+			$bridge = Bridge::create($bridgeName);
+
+			if($bridge === false || (!Bridge::isWhitelisted($bridgeName))) {
+				continue;
+			}
+
```

Comment:
The section above should be rewritten like this:

```PHP
if(!Bridge::isWhitelisted($bridgeName)) {
	continue;
}

$bridge = Bridge::create($bridgeName);

if($bridge === false) {
	continue;
}
```

or simply (although much harder to comprehend)

```PHP
if(!Bridge::isWhitelisted($bridgeName)
|| $bridge = Bridge::create($bridgeName) === false) { // This creates the $bridge object
	continue;
}
```

Reply:
That looks better yes, will add!